### DISCUSSION TITLE
Repeat row with variable

### DISCUSSION THREAD

**Discussion by MaxenceMottard** [grafana/grafana#41734] [General]
Hi, I would like to duplicate a row according to the variables of my dashboard. I have a variable named "index" which has 3 different values that I get back well. I would like to repeat this row three times because I have three different values for my variable. I try to configure my row as you can see on the screenshot but my row is not repeated.
<img width="501" alt="Capture d’écran 2021-11-16 à 11 31 10" src="https://user-images.githubusercontent.com/28975415/141969425-4ae05011-c981-47bc-9acc-6d848ca193ef.png">

Is there an issue or did I miss something?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
